### PR TITLE
fix updateConfigurationChangeResult return code and store task information

### LINKED ISSUE #1981: fix updateConfigurationChangeResult return code and store task information
- [x] Return HTTP 204 no content for updateConfigurationChangeResult instead of HTTP 200
- [x] Extend the SERVICE_CONFIGURATION_UPDATE_REQUEST table with TASK_RESULT column and store the `tasks` information from the ServiceConfigurationChangeResult object.

### PR DESCRIPTION
**Pull request by WangLiNaruto**
Fixed #1981 
image
image

image
image
image
image

### CODE REVIEW

**Review by baixinsui — COMMENTED** [review]
LGTM

**Comment by Alice1319** [comment]
LGTM

File: modules/database/src/main/java/org/eclipse/xpanse/modules/database/serviceconfiguration/update/ServiceConfigurationChangeDetailsEntity.java [hunk 1]
```diff
@@ -79,4 +80,9 @@ public class ServiceConfigurationChangeDetailsEntity implements Serializable {
     @Column(name = "STATUS")
     private ServiceConfigurationStatus status;
 
+    @Column(name = "TASK_RESULT ", columnDefinition = "json")
+    @Type(value = JsonType.class)
+    @Convert(converter = ObjectJsonConverter.class)
+    private ServiceConfigurationChangeResult taskResult;
```
Comment (swaroopar) [inline modules/database/src/main/java/org/eclipse/xpanse/modules/database/serviceconfiguration/update/ServiceConfigurationChangeDetailsEntity.java:86]: we must only store the 'tasks' from the  ServiceConfigurationChangeResult object. Not the complete ServiceConfigurationChangeResult object. 
From the ServiceConfigurationChangeResult oject, the main error and status is already stored in other columns.

**Review by swaroopar — CHANGES_REQUESTED** [review]

**Review by swaroopar — APPROVED** [review]
LGTM

**Pull request merged**